client/src/components/LoginForm/LoginContent.jsx

```diff
@@ -1,19 +1,120 @@
 import React, { Component } from 'react';
 import Section from '../Section';
 import EditText from '../EditText';
+import axios from 'axios';
 import './LoginForm.css';
 
+const LOGIN_URL = 'http://localhost:3001/users/login';
+
 class LoginContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+    };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.onLogin = this.onLogin.bind(this);
+  }
+
+  onLogin() {
+    if (this.isInputValid()) {
+      //Send Request
+      axios
+        .post(LOGIN_URL, {
+          username: this.state.username,
+          password: this.state.password,
+        })
+        .then(response => {
+          // Got good response
+          console.log(response);
```

Comment:
I think youd have to do a response.data.Response to get the actual "Authenticated" message we're sending back to you because of the new structure.. oops

Reply:
getting the email now, since it got pushed👌